doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -775,6 +775,14 @@ changing the remapped destination, ``<part>`` should be used to indicate the
 component that is being modified (see `URL Parts`_). Currently the only valid
 parts for rm-destination are QUERY, PATH, and PORT.
 
+For the query parameter, this operator takes an optional second argument,
+which is a list of query parameters to remove (or keep with ``[INV]`` modifier).
+
+::
+
+  rm-destination QUERY <comma separate list of query parameter>
+
+
```

Comment:
Perhaps to make this a bit more comprehensive, should we make [INV] available to rm-header and rm-cookie as well?